database/console/show_command.go

```diff
@@ -54,12 +54,14 @@ func (r *ShowCommand) Extend() command.Extend {
 		Category: "db",
 		Flags: []command.Flag{
 			&command.StringFlag{
-				Name:  "database",
-				Usage: "The database connection",
+				Name:    "connection",
```

Comment:
Sorry, we can keep `database` here:
 
<img width="825" alt="image" src="https://github.com/user-attachments/assets/e7c03122-f00e-4a89-ac2e-0453b68e0555" />

Reply:
Okay